src/components/TreeMenu/index.js

```diff
@@ -42,9 +41,10 @@ const TreeMenu = ({
 							/>}
 						</InputGroup>
 						<ListGroup>
-							{items.map(({ reset, ...props }) => (
+							{items.map(({ reset, ...params }) => (
 								<TreeMenuItem
 									active="false"
+									{...params}
 									{...props}
```

Comment:
@Pe5h4 I think this change is necessary because before this `props` were passed from map method and props from component were not passed there. So it is possible to rename one of the props, which is what I did.

Reply:
@aringocode So then `{...props}` does not have to be included, if I understand it correctly? Meaning - you dont have to pass the `{...props}` then

Reply:
@Pe5h4 Why they should not be included. I need to pass `resource` and `resources` to TreeMenuItem. So I can use destructuring to pull them out of the props. Would that be an acceptable solution?

```
{items.map(({ reset, ...params }) => (
    <TreeMenuItem
		active="false"
		resource={props.resource}
		resources={props.resources}
		{...params}
    />
```

Reply:
@aringocode Ok, i thought that the ...props are part of `...params`, or not?

Reply:
and what are `...params` anyway?

Reply:
@Pe5h4 props are not part of the parameters.


`...parama` are
`{
    "active": false,
    "index": 6,
    "parent": ""
}`

This photo illustrates this code. Parameters from TreeMenu component are not passed to TreeMenuItem

```
const TreeMenu = ({
	data, searchOptions, ...props
}) => {

...

{items.map(({ reset, ...props }) => (
		<TreeMenuItem
				active="false"
				{...props}
		/>
))}

```

![image](https://github.com/TeskaLabs/asab-webui/assets/65866093/d1ac9787-d1a5-431a-b8f9-89e2a3cffef6)

Reply:
Ok, thanks for clarification